Assignments/Program.cs

```diff
@@ -1 +1,46 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T13 - Strong Password
+// ---------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T15 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Check for Isogram Word:-" + "\x1B[0m");
+      Console.WriteLine ("An isogram is a word that has no duplicate letters.");
+      string output;
+      List<string> testCases = new () { "apple", "banana", "cat", "doctrine", "elephant", "fruits", "pebble", "rubiks", "yellow", "zebra" };
+      Console.WriteLine ();
+      Console.WriteLine ("\x1B[4m" + "Test Cases:" + "\x1B[0m");
+      for (int i = 0; i < testCases.Count; i++) {
+         Console.Write ($"{i + 1}. {testCases[i]} - ");
+         (output, Console.ForegroundColor) = IsIsogram (testCases[i]) ? ("Isogram", Green) : ("Not an Isogram", Red);
+         Console.WriteLine ($"{output}");
+         Console.ResetColor ();
+      }
+      Console.Write ("\nNow, let's enter a word: ");
+      string input = Console.ReadLine ().ToUpper ();
+      (output, Console.ForegroundColor) = IsIsogram (input) ? ("Isogram", Green) : ("Not an Isogram", Red);
+      Console.WriteLine ($"{output}");
+      Console.ResetColor ();
+   }
+
+   /// <summary>Returns true if input string is an isogram, otherwise false</summary>
+   public static bool IsIsogram (string input) {

```

Comment:
![image](https://github.com/vikrampg1710-tmm/Training/assets/75829713/2b706b17-5c3e-448c-99b1-1e5cf9943238)
Functionality failed. If I enter just numbers then it shows it as isogram.